src/sagas/systemIntakeSaga.ts

```diff
@@ -128,7 +131,9 @@ function postLifecycleId({ id, data }: { id: string; data: lifecycleIdData }) {
 function* issueLifecyleId(action: Action<any>) {
   try {
     yield put(issueLifecycleIdForSystemIntake.request());
-    const response = yield call(postLifecycleId, action.payload);
+    const { id, lcidPayload, actionPayload } = action.payload;
+    yield call(postSystemIntakeActionRequest, { id, ...actionPayload });
+    const response = yield call(postLifecycleId, { id, data: lcidPayload });
```

Comment:
What is this `response`? I'm seeing a `POST` and `204 No Content`. I'm not sure what I should be seeing.

If what i'm seeing is correct, we don't need to pass anything on success to the reducer.

Reply:
I think you have an older version of the backend app running. I made a change in #544 to return the intake, not just a 204 and an empty response.